### PR TITLE
resolve_eslint_NodePropertiesComponent

### PR DESCRIPTION
**Pull request by hardiknai-techm**
[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/921?feature.someFeatureFlagYouMightNeed=true)

### CODE REVIEW

File: src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx [hunk 1]
```diff
@@ -45,6 +45,7 @@ export interface NodePropertiesComponentProps {
   selectNode: (id: string) => void;
   updatePossibleVertices: () => Q.Promise<PossibleVertex[]>;
   possibleEdgeLabels: Item[];
+  //eslint-disable-next-line
```
Comment (victor-meng) [inline src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx:48]: Is there no other way to make eslint happy without disabling it?

File: src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx [hunk 2]
```diff
@@ -137,11 +135,12 @@ export class NodePropertiesComponent extends React.Component<
    * Get type option. Limit to string, number or boolean
    * @param value
    */
+  //eslint-disable-next-line
```
Comment (victor-meng) [inline src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx:138]: Same question as above.

File: src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx [hunk 3]
```diff
@@ -172,10 +171,10 @@ export class NodePropertiesComponent extends React.Component<
     ];
 
     const existingProps: ViewModels.InputProperty[] = [];
-
+    //eslint-disable-next-line
```
Comment (victor-meng) [inline src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx:174]: Same here.

Comment (hardiknai-techm) [reply src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx]: revert change

File: src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx (hunk 2 above)
Comment (hardiknai-techm) [reply src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx]: revert change

File: src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx (hunk 1 above)
Comment (hardiknai-techm) [reply src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx]: revert change

**Review by ato9000 — APPROVED** [review]

**Pull request merged**